api/v1/db/oracledb/member-dao.js

```diff
@@ -168,6 +171,30 @@ const deleteMember = async (memberId) => {
   }
 };
 
+const databaseName = string => (decamelize(string).toUpperCase());
+
+const patchMember = async (memberId, attributes) => {
+  const connection = await conn.getConnection();
+  try {
+    const joinedStringArray = _.map(attributes, (value, key) => (`${isTruthyOrZero(value) ? `${databaseName(key)} = :${key}` : ''}`));
```

Comment:
Could you break this function over multiple lines to keep it under 100 characters long?